tests/integration/tests/performance.js

```diff
@@ -1,285 +0,0 @@
-import { serveTest } from "../test-server.js";
```

Comment:
I believe this was deleted because a duplicate of `tests/integration/performance/performance.js`

Reply:
Yes exactly.